python/kserve/kserve/model_server.py

```diff
@@ -55,6 +55,10 @@
                     help="Enable docs url '/docs' to display Swagger UI.")
 parser.add_argument("--enable_latency_logging", default=True, type=lambda x: bool(strtobool(x)),
                     help="Output a log per request with latency metrics.")
+parser.add_argument("--log_config_file", default=None, type=str,
```

Comment:
@elukey Do you have an example how to pass this on the inference service yaml? I guess you need to mount this log file as ConfigMap

Reply:
@yuzisun Yes exactly something similar, it just need a file to be present on the pod, it shouldn't be too difficult. I can try to come up with an example, lemme know if it is needed :)